pkg/client/client.go

```diff
@@ -18,9 +20,12 @@ var (
 
 // Config is used to configure a Ruler Client
 type Config struct {
-	Key     string `yaml:"key"`
-	Address string `yaml:"address"`
-	ID      string `yaml:"id"`
+	Key         string `yaml:"key"`
+	Address     string `yaml:"address"`
+	ID          string `yaml:"id"`
+	TLScaFile   string `yaml:"tls_ca"`
+	TLScertFile string `yaml:"tls_cert"`
+	TLSkeyFile  string `yaml:"tls_key"`
```

Comment:
I think we can simplify this struct by embedding the [TLS struct](https://github.com/cortexproject/cortex/blob/54f10c2b1223bf16ca955ac517b55fa878d5de50/pkg/util/tls/tls.go#L15-L19) from Cortex directly, and calling `GetTLSConfig()` on it: https://github.com/cortexproject/cortex/blob/54f10c2b1223bf16ca955ac517b55fa878d5de50/pkg/util/tls/tls.go#L29

Reply:
so much simple, yes 👌  thanks!